ui/ts/pages/cluster.ts

```diff
@@ -206,6 +235,8 @@ module AdminViews {
             } else {
               let allStats: Models.Proto.NodeStatus[] = nodeStatuses.allStatuses();
               let totalStats: Models.Proto.Status = nodeStatuses.totalStatus();
+
+              axis.title = axis.titleFn(allStats);
```

Comment:
You should probably copy this into the (largely duplicated) RenderGraphs function above. Also would this mean that, for the number visualizations, the title has to be defined as a function?